### ISSUE TITLE
Encouraging users to sign in

### ISSUE THREAD

**Issue by sarahjeansweeney** [NEU-Libraries/cerberus#954]
In order to alleviate some of the confusion felt by users who don't know to sign in to see Marcom's materials, can we add a flag or visual cue to the Login button with hover over text to encourage people to sign in?

I'm open to suggestions for the design of the visual cue, but my first thought is a small, bright ribbon in the top right corner of the button. The hover over text could be: "Sign in with your myNEU username and password to see more content."

**Comment by dgcliff** [comment]
Is the problem they don't know Marcom materials is behind the sign in, or that they don't know signing in is an option?

**Comment by sarahjeansweeney** [comment]
Both, I think. They don't know Marcom's content is restricted to just faculty and staff, and they don't know they can sign in with their myNEU credentials to access more content. Marcom users are the biggest group I'm aware of who encounter this problem, but I assume this is true for a lot of users who don't know what they're missing by not signing in.

**Comment by dgcliff** [comment]
Do we have any other Northeastern website examples that prompt in a similar way? I'm curious if there's precedent for this. I'm mostly just trying to think of a pleasant way of saying "people who aren't Northeastern, you can't see some things, and never will"

**Comment by sarahjeansweeney** [comment]
Scholar OneSearch does it: 
![screenshot_2016-05-12-10-24-56](https://cloud.githubusercontent.com/assets/4663073/15218043/1556b0ee-182c-11e6-83ec-7433717b4d36.png)

**Comment by dgcliff** [comment]
Excellent. Let's copy that, that way if anyone complains we have a scapegoat :sunglasses:

**Comment by sarahjeansweeney** [comment]
I'm not worried about anyone complaining. Our user group is the Northeastern community, and they all have logins. Anyone else would be part of the general public and, as much as I would like to, we can't support the public and the NU community equally. I'd rather have a feature that makes using the DRS clearer for the primary group than worry about the how the public group will react.

**Comment by elizoller** [comment]
SOS style alert but with the text you provided
when hovering over login button
<img width="1322" alt="screen shot 2016-06-02 at 9 49 53 am" src="https://cloud.githubusercontent.com/assets/5439169/15747160/6c640af4-28a7-11e6-9053-2da7c42b0632.png">

**Comment by sarahjeansweeney** [comment]
Looks good to me.

**Comment by dgcliff** [comment]
This is fussy, but is there a possibility for a softer yellow? We use that specific flash for typically mechanical warnings (something may have gone wrong, warning etc.), whereas this feels like more of a gentle nudge. Just comparing to the SOS style.

**Comment by elizoller** [comment]
No problemo. I'll put some other screenshots up in a bit.

**Comment by elizoller** [comment]
Using the regular info alert:
<img width="1229" alt="screen shot 2016-06-06 at 12 54 52 pm" src="https://cloud.githubusercontent.com/assets/5439169/15830878/8e284cce-2be8-11e6-8bfc-4f5bf83efc54.png">
Using the brand "primary" color:
<img width="1212" alt="screen shot 2016-06-06 at 12 58 45 pm" src="https://cloud.githubusercontent.com/assets/5439169/15830879/8e283496-2be8-11e6-8b7c-76f12e3b4cb4.png">
Using a lighter yellow background and the navy blue text:
<img width="1221" alt="screen shot 2016-06-06 at 1 06 35 pm" src="https://cloud.githubusercontent.com/assets/5439169/15830877/8e241424-2be8-11e6-9804-61dbc7eb4c58.png">
Using a lighter yellow background and white text:
<img width="1218" alt="screen shot 2016-06-06 at 1 07 54 pm" src="https://cloud.githubusercontent.com/assets/5439169/15830880/8e2ac8e6-2be8-11e6-9d8c-f2735df72f09.png">
Using a lighter yellow background with darker yellow text:
<img width="1208" alt="screen shot 2016-06-06 at 1 09 49 pm" src="https://cloud.githubusercontent.com/assets/5439169/15830876/8e23bea2-2be8-11e6-9ad0-0ff94c0fc3b3.png">
Using a lighter yellow background with darker yellow text and making sign in an actual link which makes it bold and underlines it:
<img width="1203" alt="screen shot 2016-06-06 at 1 11 09 pm" src="https://cloud.githubusercontent.com/assets/5439169/15830875/8e1a3800-2be8-11e6-9fb3-c0000e2b8b08.png">

Thoughts?
I didn't think using gray would be a good idea because the nav bar is gray already.

**Comment by dgcliff** [comment]
I like number two and the last one - I think linking the "Sign in" text is a good idea. We've found before that people don't intuit the login link (I still don't know why, but there it is)

**Comment by sarahjeansweeney** [comment]
I prefer 2. Would it be overkill to make the whole notice a link to sign in?

**Comment by dgcliff** [comment]
I'd worry that people may accidentally click it in attempting to dismiss it (the x at the far right)

**Comment by sarahjeansweeney** [comment]
Fair enough. As long as the "Sign in" text is clearly a link, it should be fine.

**Comment by elizoller** [comment]
@sarahjeansweeney do you want this alert just on the homepage?

**Comment by sarahjeansweeney** [comment]
yes, homepage only please!
